docs/rules/valid-title.md

```diff
@@ -148,3 +148,45 @@ describe('foo', () => {
   test('bar', () => {});
 });
 ```
+
+## Options
+
+```ts
+interface {
+  ignoreTypeOfDescribeName?: boolean;
+  disallowedWords?: string[];
+}
+```
+
+#### `ignoreTypeOfDescribeName`
+
+Default: `false`
+
+When enabled, the type of the first argument to `describe` blocks won't be
+checked.
+
+#### `disallowedWords`
+
+Default: `[]`
+
+A string array of words that are not allowed to be used in test titles. Matching
+is not case-sensitive, and looks for complete words:
```

Comment:
```suggestion
is not case sensitive, and looks for complete words:
```
?

Reply:
Generally this [should be hyphenated](https://community.languagetool.org/rule/show/EN_COMPOUNDS?lang=en).